hikari/impl/special_endpoints.py

```diff
@@ -1289,6 +1289,14 @@ class CommandBuilder(special_endpoints.CommandBuilder):
         alias="name_localizations", factory=dict, kw_only=True
     )
 
+    _integration_types: typing.Optional[typing.Sequence[applications.ApplicationIntegrationType]] = attrs.field(
+        alias="integration_types", factory=list, kw_only=True
+    )
```

Comment:
This will never be none, as it has a factory

```suggestion
    _integration_types: typing.Sequence[applications.ApplicationIntegrationType] = attrs.field(
        alias="integration_types", factory=list, kw_only=True
    )
```